backend/app/cmd/server.go

```diff
@@ -143,7 +152,7 @@ type serverApp struct {
 
 // Execute is the entry point for "server" command, called by flag parser
 func (s *ServerCommand) Execute(args []string) error {
-	log.Printf("[INFO] start server on port %d", s.Port)
+	log.Printf("[INFO] start rest server in '%s' mode", s.SSL.Mode)
```

Comment:
not sure how this will work with "none" mode. In this case the port should has nothing to do with SSL

Reply:
I've reverted it. I don't know what to log here in static or auto mode. In rest.Run also logs on which port server is running